src/modules/theme/components/pagination/pagination.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Input, Output, EventEmitter, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'pagination',
+  templateUrl: './pagination.component.html',
+  styleUrls: ['./pagination.component.scss']
+})
+export class PaginationComponent implements OnInit {
+
+  @Input() currentPage?: number;
+  @Input() pages?: number[];
+  @Output() updatePageEvent = new EventEmitter<{ pageNumber: number }>();
+
+  constructor() {
+    this.currentPage = this.currentPage || 0;
+    this.pages = this.pages || [];
+  }
+
+  ngOnInit(): void {
+  }
+
+
+  onNextPage(): void {
+    if (this.currentPage) {
+      this.updatePageEvent.emit({ pageNumber: this.currentPage += 1 });
+    }
+  }
+
+  onPreviousPage(): void {
+    if (this.currentPage) {
```

Comment:
You should verify if the current page is above 1...

Reply:
this check is made on the html before drawing the Prev button